es/03_1_Verificando_Su_Configuracion_Bitcoin.md

```diff
@@ -0,0 +1,116 @@
+
+Si no está actualizado, pero la salida de su `getblockcount` está aumentando, no hay problema. El tiempo total de descarga puede llevar desde una hora hasta varias horas, según su configuración.
+
+## Opcional: conozca los tipos de servidor
+
+> **TESTNET vs MAINNET:** Cuando configura su nodo, usted puede elegir crearlo como un nodo Mainnet, Testnet o Regtest. Aunque este documento presupone una configuración de testnet, vale la pena comprender cómo puede acceder y utilizar los otros tipos de configuración, ¡incluso todos en la misma máquina! Pero, si es un usuario nuevo, omita esto, ya que no es necesario para una configuración básica.
+
+El tipo de configuración se controla principalmente a través del archivo ~/.bitcoin/bitcoin.conf. Si está ejecutando testnet, probablemente contenga esta línea:
+```
+testnet=1
+```
+Si está ejecutando regtest, probablemente contenga esta línea:
+```
+regtest=1
+```
+
+Sin embargo, si desea ejecutar varios tipos diferentes de nodos simultáneamente, debe dejar la marca testnet (o regtest) fuera de su archivo de configuración. Luego puede elegir si está utilizando la red principal, la red de prueba o su registro cada vez que ejecuta bitcoind o bitcoin-cli.
+
+Aquí hay un conjunto de alias que lo facilitarían al crear un alias específico para iniciar y detener el bitcoind, para ir al directorio de bitcoin y para ejecutar bitcoin-cli, para cada una de las redes principales (que no tiene indicadores adicionales), el testnet (que es -testnet), o su regtest (que es -regtest).
+
+```
+cat >> ~/.bash_profile <<EOF
+alias bcstart="bitcoind -daemon"
+alias btstart="bitcoind -testnet -daemon"
+alias brstart="bitcoind -regtest -daemon"
+
+alias bcstop="bitcoin-cli stop"
+alias btstop="bitcoin-cli -testnet stop"
+alias brstop="bitcoin-cli -regtest stop"
+
+alias bcdir="cd ~/.bitcoin/" #linux default bitcoin path
+alias btdir="cd ~/.bitcoin/testnet" #linux default bitcoin testnet path
+alias brdir="cd ~/.bitcoin/regtest" #linux default bitcoin regtest path
+
+alias bc="bitcoin-cli"
+alias bt="bitcoin-cli -testnet"
+alias br="bitcoin-cli -regtest"
+EOF
+```
+
+Para una mayor complejidad, puede hacer que cada uno de sus alias de 'inicio' use el indicador -conf para cargar la configuración desde un archivo diferente. Esto va mucho más allá del alcance de este tutorial, pero lo ofrecemos como punto de partida para cuando sus exploraciones de Bitcoin alcancen el siguiente nivel.
+
+## Resumen: verificación de la configuración de Bitcoin
+
+Antes de comenzar a jugar con bitcoin, debe asegurarse de que sus alias estén configurados, su bitcoind se esté ejecutando y sus bloques estén descargados. Es posible que también desee configurar algún acceso a configuraciones alternativas de Bitcoin, si es un usuario avanzado.
+
+## Que sigue?
+
+Continue con "Entendiendo su Configuración Bitcoin" en [§3.2: Entendiendo su configuración Bitcoin](03_2_Conociendo_Su_Configuracion_Bitcoin.md).
```

Comment:
- `Continue "Entendiendo su Configuración Bitcoin"`
- Link in LEEME.md: `[3.2: Conociendo Su Configuración Bitcoin](03_2_Conociendo_Su_Configuracion_Bitcoin.md)`